ros_controls.rolling-on-kilted.repos

```diff
@@ -0,0 +1,37 @@
+repositories:
+  ros-controls/control_msgs:
+    type: git
+    url: https://github.com/ros-controls/control_msgs.git
+    version: master
+  ros-controls/control_toolbox:
+    type: git
+    url: https://github.com/ros-controls/control_toolbox.git
+    version: ros2-master
+  ros-controls/gz_ros2_control:
+    type: git
```

Comment:
Just a curious question, does the softlinks work? or does it have to be a physical file always. Because if it works, we can also do a softlink to rolling.repos that's why

Reply:
no idea, does git itself support symlinks?

Reply:
and someday the kilted branches will change anyways

Reply:
Sure. Yes, git supports softlinks